suites/apis/rasCronjobValidate.js

```diff
@@ -0,0 +1,19 @@
+Feature('RAS cronjob validation');
+
+const { expect } = require('chai');
+const { interactive, ifInteractive } = require('../../utils/interactive.js');
+
+const TARGET_ENVIRONMENT = process.env.GEN3_COMMONS_HOSTNAME;
+
+Scenario('Validate fence-visa-update job @manual', ifInteractive(
+  async () => {
+    const result = await interactive(`
+            1. Get a new passport from /userinfo endpoint
+            2. Log into ${TARGET_ENVIRONMENT} with RAS Login (RAs user and password)
+            3. run `gen3 job run fence-visa-update`
+            4. run `gen3 psql fence`
+            5. check ga4gh_visa_v1 table for the new entry
```

Comment:
now double check the number of visas for your user again, you should have more. and the new one(s) should have different JTIs and expirations than from when you checked before

Reply:
adding step to check the visa table after the job is run for different JTI and expiration on the new visa